File: Python projects/Python Basics/conditional_statements/lab/area_of_figures.py
from math import pi
form = str(input())
if form == "square":
    length = float(input())
    area = length * length
    print(f"{area:.3f}")
elif form == "rectangle":
    length = float(input())
    width = float(input())
    area = length * width
    print(f"{area:.3f}")
elif form == "circle":
    radius = float(input())
    area = pi * radius * radius
    print(f"{area:.3f}")
elif form == "triangle":
    side = float(input())
    hight = float(input())
    area = side * hight / 2
    print(f"{area:.3f}")

File: JavaScript projects/JavaScript Basics/Conditional Statements/lab/area_of_figures.js
function areaFigures(figure, numberOne, numberTwo){
    let result;
    if (figure == 'square'){
        result = numberOne * numberOne;
    }else if (figure === 'rectangle'){
        result = numberOne * numberTwo;
    }else if (figure === 'circle'){
        result = Math.PI * numberOne * numberOne;
    }else if (figure === 'triangle'){
        result = numberOne * numberTwo / 2;
    }

    console.log(result.toFixed(3));
}

areaFigures("triangle", 4.5, 20);